jenkins/dockerfiles/gcc-8_arm64/Dockerfile

```diff
@@ -31,4 +31,5 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
    libcap-dev:arm64 \
    libcap-ng-dev:arm64 \
    libelf-dev:arm64 \
-   libpopt-dev:arm64
+   libpopt-dev:arm64 \ 
+   libc6-dev-amd64-cross:amd64
```

Comment:
This looks strange, it's an x86 package to cross-compile x86.  If you need to cross-compile x86 (amd64) on arm64, then you need `libc6-dev-amd64-cross:amd64`.  If it's for building host tools, then you need `libc6-dev` (without `:arm64`).